src/common/mockFactory.ts

```diff
@@ -901,4 +910,8 @@ export default class MockFactory {
                 return StorageType.Other;
         }
     }
+
+    private static randomNumber(min: number, max: number) {
```

Comment:
Not that this matters much since it's so simple, but this already exists in the `utils` file

Reply:
Thanks - will convert to use that method.